apps/src/sites/studio/pages/levels/_curriculum_reference.js

```diff
@@ -0,0 +1,33 @@
+window.curriculumReference = {
+  onload: onload
+};
+
+/**
+ * Called on load of our iframe. This hides the spinner, shows the iframe, and
+ * sets up a method to keep our iframe correctly sized.
+ */
+function onload() {
+  const iframe = document.getElementById('curriculum-reference');
+  // For some reaosn onload is called multiple times in Chrome. We only want to
+  // start our interval once.
+  if (iframe.style.display !== 'none') {
+    return;
+  }
+
+  // Make iframe visible, hide spinner
+  iframe.style.display = '';
+  document.getElementById('iframe-loading').style.display = 'none';
+
+  // Our iframe can resize for various reasons (initial load, images loaded on
+  // iframe page, window resized). We want to keep the size of the iframe in
+  // sync with the contents of the firame.
+  setInterval(() => {
+    resizeIFrame(iframe);
+  }, 200);
```

Comment:
Ah, sorry, I was suggesting using capture = true to get called each time a sub-resource loads:
```js
window.addEventuListener("load", callback, true):
```
It looks like we're doing this to try to make the frame seamless and use the regular page scrollbar.  Could we keep the header in place and make the iframe take up the rest of the page?  (Using the iframe scrollbar instead of the top-level scrollbar.)